### DISCUSSION TITLE
Unexpected error

### DISCUSSION THREAD

**Discussion by WoofahRayetCode** [liquidctl/liquidctl#470] [General]
Hello,

I just recently found this program when I was searching for programs to use my corsair stuff on Manjaro. I followed the readme as best I could. I am running into this error.

`ERROR: unexpected error with Corsair Commander Core (experimental): AssertionError('response does not match command')`

I am not sure what I can do to fix it from the several things mentioned in the readme. I am not sure what other info to include to better get help. I am still a big Linux noob. I am not sure if I overlooked something.

Any help is appreciated. This gets me one step closed to leaving Windows.

**Comment by jonasmalacofilho** [comment]
This is caused by a change in the latest firmware/protocol of that particular device.  We're tracking the bug with issue #464.

**Reply by WoofahRayetCode** [reply]
> This is caused by a change in the latest firmware/protocol of that particular device. We're tracking the bug with issue #464.

Thank you for your quick reply. I subbed to that bug report and will keep an eye on its progress.
